### ISSUE TITLE
Masterserver ipv4/ipv6

### ISSUE THREAD

**Issue by DenMSC** [Qfusion/qfusion#441]
When a server sends a heartbeat to masterservers, it *only* sends to the ipv6 of the domains in "masterservers" cvar, if they are available. (atm only dpmaster.deathmask.net)

Can be manually fixed by adding the raw ipv4, but that's not very flexible.

The heartbeat should be sent to both ipv4 and ipv6.

**Comment by viciious** [comment]
Yeah, right but I’m not taking feature request or bug reports for the engine anymore. Feel free to submit a pull request.

**Comment by Cossid** [comment]
For a non-engine change, you can use multiple masterserver cvars, but this will be dependent on having hosts specific to the protocol you need.

dpmaster.deathmask.net (bound to both ipv4 and ipv6) does have alternative specific bindings which you can use directly or cname to:
ipv4.dpmaster.deathmask.net
ipv6.dpmaster.deathmask.net (also bound to both ipv4 and ipv6, but if the client is ipv6, it will prefer ipv6, and ipv4 is a safe fallback).
